### PR TITLE
Symfony 5 Suport

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

**Comment by dantleech** [comment]
@alexander-schranz the Process component was just an example -- only reason for reverting that time was because the tests wern't running so we could be sure?

Anyway -- let's see if the test pass this time ...

File: composer.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
     "description": "Shell for PHPCR",
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/console": "~2.8|~3.0|~4.0",
+        "symfony/console": "~2.8|~3.0|~4.0|~5.0",
```
Comment (dbu) [inline composer.json]: can we use the `^` syntax? and i'd raise to `^3.4 | ^4.3 | ^5.0` to not have to deal with super old legacy. people stuck on that who don't use the shell stand alone won't care about the symfony 5 support anyways.

**Review by dbu — COMMENTED** [review]
why do we run phpunit at all when there are no unit tests? or is it misconfigured and there would be tests.

regarding the jackrabbit errors, you might want to try something like this https://github.com/doctrine/phpcr-odm/pull/800/files#diff-07732eacf103d175673d25f4f64bd189 to install a more recent jackrabbit version

File: composer.json (hunk 1 above)
Comment (dantleech) [reply composer.json]: just legacy, will update

**Pull request merged**